models/discount-code/src/transformers.ts

```diff
@@ -30,6 +30,7 @@ const transformers = {
       'cartDiscounts',
     ],
     addFields: ({ fields }) => ({
+      applicationCount: fields.applicationVersion,
```

Comment:
`applicationVersion` = `applicationCount` ?😕

I would expect this to be on the [generator](https://github.com/commercetools/test-data/blob/f7fe5b7f9dedfca6a2612ce09e515a0125572074/models/discount-code/src/generator.ts#L15), but can't see the prop on the [docs](https://docs.commercetools.com/api/projects/discountCodes#discountcode) either.
Is this going to be an only graphql field?

Reply:
> `applicationVersion` = `applicationCount` ?😕
> 
> I would expect this to be on the [generator](https://github.com/commercetools/test-data/blob/f7fe5b7f9dedfca6a2612ce09e515a0125572074/models/discount-code/src/generator.ts#L15), but can't see the prop on the [docs](https://docs.commercetools.com/api/projects/discountCodes#discountcode) either. Is this going to be an only graphql field?

I only found the property in GraphQL docs. 
I have no idea if `applicationVersion` = `applicationCount`, so, I've added a commit where `applicationCount` is randomly generated.